app/styles/_openshift-logos-icon.less

```diff
@@ -191,6 +191,6 @@
 .icon-apache:before {
   content: "\f138";
 }
-.icon-casadra:before {
+.icon-casadra:before, .icon-cassandra:before {
```

Comment:
Might be worth a comment here not to remove `icon-casadra` for backwards compatibility.